core/templates/pages/exploration-player-page/exploration-player-page.mainpage.html

```diff
@@ -14,13 +14,6 @@
         }
       }
     </style>
-    <!-- Prefetch files to enable the MathExpressionInput interaction to work
```

Comment:
Why are these deleted here? @iamprayush is removing these in #10361 and it seems more relevant to that PR than this?

Reply:
According to my milestone this PR should remove Mathjax completely from Exploration player.
But if you insist I can remove this change and let @iamprayush do it.
cc @DubeySandeep .

Reply:
IMO, my PR should make this change, so that in the future if we ever wanted to revert the old math interaction deprecation(which is done in my PR), these lines would be re-added accordingly.

Reply:
Oh ok thanks, then I'll remove these changes.

Reply:
Considering this code is also being used by `MathExpressionInput interaction` we are leaving it to @iamprayush to decide when to remove this. @iamprayush, FYI math-expression in HTML doesn't need this library for rendering math-expressions in the learner's page, so feel free to remove this once `MathExpressionInput interaction` doesn't need this!